upcoming-release-notes/1899.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Bugfix
```

Comment:
I'd tag this as a feature personally!

Reply:
Oops totally my bad! I was in bugfixing mode in my mind 😆